app/assets/stylesheets/topbar.scss

```diff
@@ -1,13 +1,20 @@
 @import 'color';
 
 
-.top-bar {
+div.top-bar.normal {
+  background-image: image-url('light-wood-texture.jpg');
+  background-size: 100%;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, padding